HW02_B_Mimic_starter.Rmd

```diff
@@ -91,6 +106,14 @@ The above graph lets you see some colobrlind friendly palettes. For the graph be
 Now using the above mpg dataset, make this graph
 
 ```{r graph 4 code}
+ggplot(mpg, aes(class, cty))+
+  
+  scale_color_brewer(palette="Set2")+
+  scale_fill_brewer(palette="Set2")+
+  geom_dotplot(binaxis='y', dotsize=0.5, stackdir='center', aes( fill=class, color=class))+

```

Comment:
This is an interesting implementation, which creates well-aligned dots in y-direction comparing to the random style using jittering.